### PR TITLE
Chef 2126 create cli subcommand to start verification as systemd service ok

### PR DESCRIPTION
**Pull request by vivekshankar1**
### :chains: Related Resources
https://chefio.atlassian.net/browse/CHEF-2126

### :white_check_mark: Checklist

**All PRs** must tick these:

- [ ] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [ ] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [ ] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [ ] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [ ] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [ ] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [ ] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### :camera: Screenshots, if applicable

https://progresssoftware.sharepoint.com/:v:/s/ChefCoreC/EbFG2ggblUVMs45bCOSYu8MB5AvRWO3B5JDZqc9OPK7t_g?e=7Yfhfs

### CODE REVIEW

File: components/automate-cli/pkg/verifyserver/utils/fiberutils/logconfig.go [hunk 1]
```diff
@@ -12,7 +12,7 @@ import (
 func GetLogConfig(log logger.ILogger) (lc middleware.LoggerConfig) {
 	lc = middleware.LoggerConfig{
 		TimeFormat: time.RFC3339,
-		TimeZone:   "Asia/Kolkata",
+		TimeZone:   "UTC",
```
Comment (vivek-yadav) [inline components/automate-cli/pkg/verifyserver/utils/fiberutils/logconfig.go:15]: Give option to override using Env Var

Comment (vivekshankar1) [reply components/automate-cli/pkg/verifyserver/utils/fiberutils/logconfig.go]: Done

File: components/automate-cli/pkg/verifysystemdcreate/systemdcreatesvc.go [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+	systemdCreateUtils SystemdCreateUtils,
+	binaryDestinationFolder string,
+	systemdLocation string,
+	writer *cli.Writer) (*CreateSystemdService, error) {
+	if binaryDestinationFolder == "" {
+		return nil, errors.New("Binary destination folder cannot be empty")
+	}
+	if systemdLocation == "" {
+		return nil, errors.New("Systemd location cannot be empty")
+	}
```
Comment (Dmaddu) [inline components/automate-cli/pkg/verifysystemdcreate/systemdcreatesvc.go:57]: Any reason for not validating `systemdCreateUtils` and  `writer`?

**Review by Dmaddu — CHANGES_REQUESTED** [review]

File: components/automate-cli/pkg/verifysystemdcreate/systemdcreatesvc.go (hunk 2 above)
Comment (vivekshankar1) [reply components/automate-cli/pkg/verifysystemdcreate/systemdcreatesvc.go:75]: added nil check

File: components/automate-cli/pkg/verifyserver/utils/fiberutils/logconfig.go [hunk 3]
```diff
@@ -1,6 +1,7 @@
 package fiberutils
```
[the remark this answers is not in the document]
Comment (vivekshankar1) [reply components/automate-cli/pkg/verifyserver/utils/fiberutils/logconfig.go:1]: added

File: components/automate-cli/pkg/verifyserver/utils/fiberutils/logconfig.go [hunk 4]
```diff
@@ -9,10 +10,17 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+const VERIFY_SERVER_TIMEZONE = "VERIFY_SERVER_TIMEZONE"
+
 func GetLogConfig(log logger.ILogger) (lc middleware.LoggerConfig) {
+	timeZone := "UTC"
+	envTimeZone := os.Getenv(VERIFY_SERVER_TIMEZONE)
+	if envTimeZone != "" {
+		timeZone = envTimeZone
```
[the remark this answers is not in the document]
Comment (vivekshankar1) [reply components/automate-cli/pkg/verifyserver/utils/fiberutils/logconfig.go]: done

File: components/automate-cli/cmd/chef-automate/verify.go [hunk 5]
```diff
@@ -143,6 +171,22 @@ func (v *verifyServeCmdFlow) runVerifyServeCmd(cmd *cobra.Command, args []string
...
+func verifySystemdCreateFunc() func(cmd *cobra.Command, args []string) error {
+	return func(cmd *cobra.Command, args []string) error {
+		c := verifySystemdCreateFlow{}
+		return c.runVerifySystemdCreateCmd(cmd, args)
+	}
+}
+
+func (v *verifySystemdCreateFlow) runVerifySystemdCreateCmd(cmd *cobra.Command, args []string) error {
+
+	createSystemdServiceWithBinary, err := verifysystemdcreate.NewCreateSystemdService(verifysystemdcreate.NewSystemdCreateUtilsImpl(), BINARY_DESTINATION_FOLDER, SYSTEMD_PATH, writer)
```
[the remark this answers is not in the document]
Comment (vivekshankar1) [reply components/automate-cli/cmd/chef-automate/verify.go]: done

**Review by Dmaddu — APPROVED** [review]

**Pull request merged**